Mathlib/Logic/Equiv/PartialEquiv.lean

```diff
@@ -560,6 +560,19 @@ theorem ofSet_coe (s : Set α) : (PartialEquiv.ofSet s : α → α) = id :=
 theorem ofSet_symm (s : Set α) : (PartialEquiv.ofSet s).symm = PartialEquiv.ofSet s :=
   rfl
 
+/-- `Function.const` as a `PartialEquiv`.
+  It consists of two constant maps in opposite directions. -/
+@[simps]
+def single (a : α) (b : β) : PartialEquiv α β where
+  toFun := Function.const α b
+  invFun := Function.const β a
+  source := {a}
+  target := {b}
+  map_source' := fun _ _ ↦ by rfl
```

Comment:
You can write this as
```suggestion
  map_source' _ _ := by rfl
```
(or even just `rfl`), right? Please also make the analogous change to the other fields.